[chapter] Influence of cell-level EQE variation on PV performance (#1)
sandialabs/pv-atlas: website/source/chapter/cell-eqe-influence.rst

performance resulting in mismatch losses.  While manufacturing inconsistencies
between cells are often cited as a cause of intramodule mismatch losses, they
are typically overlooked in the existing body of research.

One way to study the inherent differences between cells is through analyzing
their external quantum efficiency (EQE).  A cell's EQE curve represents the
efficiency with which incident photons of different wavelengths generate a
photocurrent.  Although this spectral effect on PV performance has been studied
at the module level, no studies have analysed the spectral effect at the cell
level and its potential role as a driver of mismatch losses at the module
level.  

To investigate the role of cell-to-cell variation on module performance, the
following questions are proposed:

#. How does the EQE vary between cells?
#. What is the impact on PV performance?
#. Does the performance impact vary seasonally or geographically?

Methodology
----------
EQE curves for each cell within each of 11 PV modules were measured by the Cell
Measurement and Performance group at the National Renewable Energy Laboratory.

> These data are in conjunction with simulated meteorological data, including

Reviewer: ```suggestion
These data are in conjunction with one year of simulated 30-minute meteorological data, including
```

spectral irradiance, spanning the contiguous United States at a resolution of

> around 2° longitude/latitude (around 20km) thus covering around 21,000

Reviewer: ```suggestion
around 0.2° longitude/latitude (around 20km) thus covering around 21,000
```

individual locations.

The PV devices studied cover a range of technologies, including
monocrystalline silicon (monoSi) and polycrystalline silicon
(polySi), and a range of cell architectures including Al-BSF (Aluminium Back
Surface Field), IBC (Interdigitated Back Contact), PERC (Passivated Emitter

> and Rear Contact), PERT (Passivated Emitter Rear Totally Diffused), and HIT
> (Heterojunction with Intrinsic Thin Layer). IEC characterisation data as well

Reviewer: ```suggestion
and Rear Contact), PERT (Passivated Emitter Rear Totally Diffused), and SHJ
(Silicon heterojunction). IEC 61853-1 characterisation data as well
```

as STC and field performance reports of the modules can be found in XXXX.

> XX refer to data section or add references directly here? XX

Reviewer: I'd say this page doesn't need this info; interested readers can go to the paper for full detail.

Author: Changed to
```
IEC 61853-1 characterisation data and field
performance reports of the modules can be found in [X].
```
[X] is the reference to the paper
Or just delete this part all together?

Author: > [X] is the reference to the paper Or just delete this part all together?

Deleted all together. Marking as resolved

The spectral mismatch factor, $M$ is used to quantify the change in PV
performance due to changes in the spectrum.  $M$ represents the ratio of the
usable fraction (UF) of measured irradiance to the usable fraction of the
reference irradiance XX IEC reference? XX, and is calculated using the spectral

> response (SR) and spectral irradiance data as follows:

Reviewer: I think this paragraph and the ones following are getting too far into the weeds.  I think one high-level paragraph is good here, communicating that we're looking at the difference between "worst cell" and "average cell".

level.  

To investigate the role of cell-to-cell variation on module performance, the
following questions are posed:

#. How does the EQE vary between cells?
#. What is the impact on PV performance?
#. Does the performance impact vary seasonally or geographically?

Methodology
-----------

EQE curves for each cell within each of 11 PV modules were measured by the Cell
Measurement and Performance group at the National Renewable Energy Laboratory.
These data are analysed in conjunction with one year of simulated 30-minute
meteorological data, including spectral irradiance, spanning the contiguous
United States at a resolution of around 0.2° longitude/latitude (around 20km).
At this resolution, the dataset covers around 21,000 individual locations.

The PV devices studied cover a range of technologies, including monocrystalline
silicon (monoSi) and polycrystalline silicon (polySi), and a range of cell
architectures including Al-BSF (Aluminium Back Surface Field), IBC
(Interdigitated Back Contact), PERC (Passivated Emitter and Rear Contact), PERT
(Passivated Emitter Rear Totally Diffused), and SHJ (Silicon heterojunction).

> To quantify the impact of intradmodule EQE variation on PV device performance,

Reviewer: ```suggestion
To quantify the impact of intramodule EQE variation on PV device performance,
```

There are four forms of EQE variation among the devices, classified by this
study as follows:

#. Negligible
#. Amplitude
#. Wavelength
#. Mixed

The first type of variation is characterized by a tight distribution with a
mean absolute deviation of between 1% and 3% that is largely independent of
wavelength.  Examples include the CSPoly270 and Itek360 modules. The EQE
variation for the Panasonic325 module is also independent of wavelength, but a
significantly greater amplitude-dominant variation is present with a relatively
stable MAD of around 5%.  This relatively uniform variation in EQE can be
caused by a mixture of internal factors such as parasitic losses, and external
factors such as reflection leading to variable illumination from cell to cell
for each measurement.  The third category is characterized predominantly by
wavelength-dependent variation, such as in the case of the Qpoly280 device. 
Finally, devices such as the Mission300 module exhibit a mixture of
wavelength-dependent and amplitude variation, thus falling into the fourth
category.  The cell-level EQE curves for all 11 modules are shown in Figure 1.

.. figure:: ../_static/eqe_all_mean.png

> Figure 1. Cell EQE, median EQE, and MAD for the PV devices.

Reviewer: I think if you indent this text to the same level as the `f` in `figure`, it will be interpreted as a figure caption and rendered as such, rather than normal text following the figure.

(a) CSpoly270. (b) Itek360. (c) Jinko260. (d) LG320. (e) LG400.
(f) Mission300. (g) Panasonic325. (h) Qmono300. (i) Qpoly280.

> (j) Solaria400. (k) Trina260.y foo diagram

Reviewer: ```suggestion
(j) Solaria400. (k) Trina260.
```

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Figure 2 shows the annual energy difference between the potential
non-mismatched output, as determined by the mean EQE, and the realistic module
output, as determined by the limiting cell. The overall energy difference, ΔE,
is relatively low for all modules, rarely exceeding ±0.20%.  Nonetheless, there
is a link between the scale of loss and the type of EQE variation.  Modules
with greatest ΔE are also those whose intramodule EQE variation is
characterized by a wavelength dependence (category 3) or mixed variation
(category 4). Variation in amplitude is likely to result primarily from
systematic measurement error, for example due to variable cell illumination,
and its effect is removed through an inherent normalization in the calculation
of M.

.. figure:: ../_static/monthly-seasonality-boxplots.png

Figure 2. Seasonality breakdown for the 11 modules. Subplot (a)
presents the monthly energy difference, for each location, for the
Mission300 module. Subplot (b) shows the energy difference between the
minimum and maximum ΔE months, for each location, for all 11 modules.
In all plots, the horizontal bar represents the median, while the

> triangle represents the mean.

Author: @kandersolar would moving this to the section below resolve your comment?

Does the performance impact vary seasonally or geographically?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Could the low annual ΔE be a result of month-to-month variations cancelling
each other out? Taking the Mission300 module as an example, Figure 2 shows the
monthly breakdown ΔE values.  The range in each month represents the range of
ΔE across the contiguous USA while the triangle and horizontal bar represent
the mean and median ΔE, respectively.  On a monthly scale, the energy
difference is more significant.  The range from month to month is almost double
the annual energy difference, with an approximately 50:50 split between
positive and negative ΔE months.

Since ΔE can be positive or negative, besides reducing the annualized ΔE, this
phenomenon also indicates that the driving cell under the prevailing spectrum
can yield both an energy gain or loss with respect to performance driven by the

> mean EQE cell in the absence of intramodule EQE variation.

Reviewer: Seems like this arrangement of paragraphs doesn't match the headers.  The previous subsection also talked about seasonality, for example.

In terms of geography, ΔE has a range of 0.46% across all locations, which is
over four times the annualized ΔE.  Another phenomenon related to location is
the fact that the annual ΔE values are notably larger in the south/south east
USA than in the western states. This effect is particularly significant in
coastal states such as Florida, where the subtropical climate is driven by high
levels of humidity that are known to increase the spectral influence on PV

> performance :cite:`marion2014analysis`.

Reviewer: Unfortunately using `:cite:` messes up bibliographies elsewhere in the website.  It's on my list to fix, but for now let's remove the `:cite:`s and just list references at the bottom of this page as needed.

Author: by manually adding a number, name/year in brackets, or nothing?

Reviewer: Like this: https://github.com/sandialabs/pv-atlas/blob/main/website/source/chapter/climatic-influence-plr.rst?plain=1#L111-L120

Include the keys in the `.. bibliography::`, but don't `:cite:` them anywhere in the page.  If you need to reference them in the text, let's just do hardcoded (Name, Year) for now.

Author: So not this? https://github.com/sandialabs/pv-atlas/blob/bf305322c5e630fa34ad5f9cb039b6b4b966b275/website/source/chapter/climatic-influence-plr.rst?plain=1#L54-L55

.. map-widget:: 
   :colorscale_min: -1
   :colorscale_max: +1
   :colorscale_name: RdBu
   :short_description: ΔE [%]
   :layers_title: Module:

    cell-level-smm/CSmono275_annual_US_2022.tiff : CSmono275
    cell-level-smm/CSpoly270_annual_US_2022.tiff : CSpoly270
    cell-level-smm/Itek360_annual_US_2022.tiff : Itek360
    cell-level-smm/Jinko260_annual_US_2022.tiff : Jinko260
    cell-level-smm/LG320_annual_US_2022.tiff : LG320
    cell-level-smm/LG400_annual_US_2022.tiff : LG400
    cell-level-smm/Mission300_annual_US_2022.tiff : Mission300
    cell-level-smm/Panasonic325_annual_US_2022.tiff : Panasonic325
    cell-level-smm/Qpoly280_annual_US_2022.tiff : Qpoly280
    cell-level-smm/Qmono300_annual_US_2022.tiff : Qmono300
    cell-level-smm/Solaria400_annual_US_2022.tiff : Solaria400

>     cell-level-smm/Trina260_annual_US_2022.tiff : Trina260

Reviewer: This seems like too many layers to me.  Maybe cut it down to four layers, with one representative for each of the four categories of EQE variation?  

Note that all 11 datasets will still be available in the listing at the bottom of the page for anyone who wants to dig into the other modules.

    
Figure 3. Annual energy difference, ΔE , comparing P2 (mean EQE) and
P1 (limiting EQE) over the course of the year for all 11 modules
across the contiguous USA. Across all locations, the annualized ΔE is
typically no greater than ± 0.2%. Five modules experience an annual
energy loss, two experience an annual energy gain, and four exhibit no
significant annualized performance variation due to cell-level EQE
variation.

Discussion
----------

While the absolute annual ΔE is restricted to only around 0.1%–0.2%, on a

> montlhly basis the uncertainty due to cell-level EQE variation increases to

Reviewer: ```suggestion
monthly basis the uncertainty due to cell-level EQE variation increases to
```
